sys/sys/link_elf.h

```diff
@@ -110,6 +110,14 @@ int _rtld_addr_phdr(const void *, struct dl_phdr_info *);
 int _rtld_get_stack_prot(void);
 int _rtld_is_dlopened(void *);
 
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(__aarch64__)
+#include <machine/c18n.h>
+void *dl_c18n_get_trusted_stk(void);
```

Comment:
Please just spell stack in full in the public API, you don't need to save two letters